spec/GraphQLOverHTTP.md

```diff
@@ -746,7 +746,7 @@ well-formed _GraphQL response_ since the source of the response may not be the
 server but instead some intermediary such as API gateways, proxies, firewalls,
 etc.
 
-# Non-Normative Notes
+# Non-normative notes
```

Comment:
Thanks for that 🙏 I was looking at the rendered view. Didn't realize the translation was made automagically 👍